content/company-info-and-process/policies/access-control-policy.md

```diff
@@ -2,69 +2,71 @@
 
 **Policy Owner**: Tech Ops
 
+**Policy Owner & Approver:** VP Operations
```

Comment:
@dadlerj @maureenlsourcegraph This was at the bottom of the page, but seems to contradict the line above that declares tech-ops as the policy owner. Who is the owner?

Reply:
I believe we're good to remove VP Operations as owner and leave Tech Ops.